gradyrobbins/fifty
# Learn More tab - feature improvements

# Description
Learn More tab:  a clickable map now renders fun facts about each state that is clicked onto the DOM.  


## Type of change

Please delete options that are not relevant.



# Testing Instructions for Change Made
click the "learn more" tab upon login;  click any state to see fun facts renders onto the DOM.  
Note:  database.json was updated with "abbr" property enabling this.usas.find(e =>e.abbr ===usas.abbr)

# Checklist:
